gunicorn/workers/gtornado.py

```diff
@@ -105,9 +105,12 @@ def run(self):
         # instance of tornado.web.Application or is an
         # instance of tornado.wsgi.WSGIApplication
         app = self.wsgi
-        if not isinstance(app, tornado.web.Application) or \
-           isinstance(app, tornado.wsgi.WSGIApplication):
-            app = WSGIContainer(app)
+
+        # Patch for tornado5
```

Comment:
nit: comment not quite correct here. You could just remove it.

Reply:
oops